### DISCUSSION TITLE
发现两个渲染的问题

### DISCUSSION THREAD

**Discussion by zhuhaisto** [tamlok/vnote#2184] [Ideas]
![图片](https://user-images.githubusercontent.com/108032980/178768200-155c34af-1706-44ed-b7d0-46b64aa7e08e.png)
1. 这个好像不会自动延申 >
![图片](https://user-images.githubusercontent.com/108032980/178768539-ddb69985-c5b9-4a5f-ac8a-974ad5d5ed5c.png)
1. 从序号后第一个开始选中 用删除线没得反应
2. 但是如果用手打删除的标签就可以

还有像这种 在 序号前 包含完整的打删除线
![图片](https://user-images.githubusercontent.com/108032980/178768375-93a74ab7-78dd-4043-8151-0d6e647eb275.png)
这个渲染好像他自己变成子元素了

**Comment by tamlok** [comment]
列表前最好有一个空行
